sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -349,6 +349,10 @@ public EventParticipantDto saveEventParticipant(@Valid EventParticipantDto dto,
 		return toPseudonymizedDto(entity, pseudonymizer);
 	}
 
+	public List<EventParticipantDto> getEventParticipantsThatAttendedByTwoPersonsTogether(List<String> personIdList) {
```

Comment:
This method should return EventParticipantSelectionDto-s

Reply:
if the EventParticipantSelectionDto is used then a new dto mapper should be also used or alter the query to use the EventParticipantSelectionDto which will lengthen it